code/modules/projectiles/guns/projectile/battle_rifle/boltgun.dm

```diff
@@ -146,8 +146,8 @@
 	force = WEAPON_FORCE_DANGEROUS // weaker than novakovic, but with a bayonet installed it will be slightly stronger
 	armor_divisor = ARMOR_PEN_GRAZING
 	caliber = CAL_SRIFLE
-	damage_multiplier = 1.7
-	penetration_multiplier = 0.7
+	damage_multiplier = 1.8
```

Comment:
If the Hornet can have Damage Multiplier 2.0 this big two-handed gun can have Damage Mult 2.0.

Reply:
naturally I would agree, however whoever before me added all the guns we have now, madde the revolvers vastly better than the bolt guns. the hornet is balance in the revolver category, the kadmin in the bolt gun category. do note that the consul does more damage than the novak/mosin.

I did not change this balance not now and neither when the armor rework was done (as it was not the intention)